packages/atomic/src/components/search/result-template-components/atomic-quickview-sidebar/atomic-quickview-sidebar.tsx

```diff
@@ -0,0 +1,140 @@
+import {Result, TermsToHighlight} from '@coveo/headless';
+import {Fragment, FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+import ArrowDown from '../../../../images/arrow-bottom-rounded.svg';
+import ArrowUp from '../../../../images/arrow-top-rounded.svg';
+import MinimizeIcon from '../../../../images/menu.svg';
+import {Checkbox} from '../../../common/checkbox';
+import {
+  getWordsHighlights,
+  QuickviewWordHighlight,
+} from '../quickview-word-highlight/quickview-word-highlight';
+
+export interface QuickviewSidebarProps {
+  iframe: HTMLIFrameElement;
+  termsToHighlight: TermsToHighlight;
+  i18n: i18n;
+  highlightKeywords: boolean;
+  onHighlightKeywords: (doHighlight: boolean) => void;
+  minimize: boolean;
+  onMinimize: (minimize: boolean) => void;
+  result: Result;
+}
+
+export const QuickviewSidebar: FunctionalComponent<QuickviewSidebarProps> = (
+  props
+) => {
+  const {iframe, termsToHighlight, highlightKeywords, minimize} = props;
+
+  const words = getWordsHighlights(iframe, termsToHighlight);
+
+  const minimizeButton = (
+    <MinimizeButton {...props} wordsLength={Object.values(words).length} />
+  );
+
+  return (
+    <div
+      class="p-4 border-r border-neutral h-full"
+      style={{backgroundColor: 'var(--atomic-neutral-light)'}}
+    >
+      {minimize && minimizeButton}
+      <div class="flex items-center">
+        <HighlightKeywordsCheckbox {...props} />
+        {!minimize && minimizeButton}
+      </div>
+
+      {highlightKeywords && !minimize && (
+        <KeywordsNavigation {...props} words={words} />
+      )}
+    </div>
+  );
+};
+
+const MinimizeButton: FunctionalComponent<
+  Pick<
+    QuickviewSidebarProps,
+    'i18n' | 'minimize' | 'onMinimize' | 'highlightKeywords'
+  > & {wordsLength: number}
+> = ({i18n, minimize, onMinimize, highlightKeywords, wordsLength}) => (
+  <atomic-icon-button
+    buttonStyle="text-transparent"
+    class={`w-fit ${minimize ? '' : 'ml-auto'}`}
+    icon={MinimizeIcon}
+    labelI18nKey="minimize"
+    tooltip={i18n.t('minimize')}
```

Comment:
Instead of just `minimize` as an aria-label, I think we should describe what component we're minimizing.

Also, I think we should avoid the wording of "minimize", since screen readers would read it a bit weirdly (e.g., "Minimize keywords navigation, expanded").

One solution which would also be relevant for the tooltip would be to use "toggle" instead:
> Toggle keywords navigation

Reply:
Similar as above, I will address accessibility issues in a different PR.